data/tribes/initialization/empire/units.lua

```diff
@@ -1147,24 +1155,43 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_warehouse",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Warehouse
+            lore = pgettext("empire_building", "‘Bis dat, qui cito dat, nil dat, qui munera tardat.’ (Who gives promptly, gives twice, who gives late, gives nothing.)"),
+            -- TRANSLATORS: Lore author helptext for an empire warehouse: Warehouse
+            lore_author = pgettext("proverb written over the doors of a warehouse"),
             -- TRANSLATORS: Purpose helptext for an empire warehouse: Warehouse
             purpose = pgettext("empire_building", "Your workers and soldiers will find shelter here. Also stores your wares and tools.")
          }
       },
       {
          name = "empire_port",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire warehouse: Port
+            lore = pgettext("empire_building", "‘Navigare necesse est.’ (To sail is necessary.)"),
```

Comment:
@klaus-halfmann I'm pretty sure this is correct.

Reply:
well then keep it.